src/components/queryEditor.tsx

```diff
@@ -179,21 +183,24 @@ function AssetTab(props: SelectedProps & { datasource: CogniteDatasource }) {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [current.externalId, query.assetQuery.withRelationships]);
   return (
-    <div className="gf-form-inline">
-      <div className="gf-form">
-        <InlineFormLabel htmlFor={`asset-select-dropdown-${query.refId}`} width={6}>Asset Tag</InlineFormLabel>
+    <InlineFieldRow>
+      <InlineField
+        label="Asset Tag"
+        labelWidth={14}
+        tooltip="Search asset by name/description"
```

Comment:
does the `InlineField` no longer require the `htmlFor` prop?

Reply:
no, it uses if there is inputId/id on the child element, then label would be tied to it https://developers.grafana.com/ui/latest/iframe.html?viewMode=docs&id=forms-inlinefield--docs&globals=&args=#inlinefield